app/models/car.rb

```diff
@@ -0,0 +1,9 @@
+class Car < ApplicationRecord
+  has_many :users, through: :reservations, dependent: :destroy
+  has_many :reservation, dependent: :destroy
```

Comment:
It's more systematic to have `:reservations` come before `:users`, since `:users` relationship is through `:reservations`